[BUG] CI test fails: TypeError: LlavaProcessor: got multiple values for keyword argument 'images'

**Describe the bug**
CI test fails: TypeError: LlavaProcessor: got multiple values for keyword argument 'images'
> FAILED tests/test_models.py::ModelTests::test_transformers_message_vl_no_tool - TypeError: LlavaProcessor:
> ...
>  got multiple values for keyword argument 'images'


**Error logs (if any)**
```python
 E           {
E             "image_token": "<image>",
E             "num_additional_image_tokens": 0,
E             "patch_size": 14,
E             "processor_class": "LlavaProcessor",
E             "vision_feature_select_strategy": "full"
E           }
E            got multiple values for keyword argument 'images'

.venv/lib/python3.10/site-packages/transformers/processing_utils.py:1383: TypeError
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -60,7 +60,7 @@ telemetry = [
 ]
 transformers = [
   "accelerate",
-  "transformers>=4.0.0",
+  "transformers>=4.0.0,<4.49.0",
   "smolagents[torch]",
 ]
 all = [

